docs-conceptual/azps-8.2.0/migrate-az-9.0.0-servicebus.md

```diff
@@ -187,13 +222,15 @@ Use Set-AzServiceBusNetworkRuleSet to add/remove multiple IP/ virtual network ru
 - Parameter `-EnableBatchedOperations` would be renamed to `-EnableBatchedOperation`.
 - Input type of parameters `-DefaultMessageTimeToLive`, `-AutoDeleteOnIdle`, `-DuplicateDetectionHistoryTimeWindow` has been changed from System.String to System.Timespan. Hence, ISO 8601 format for timespan can NO longer be fed as input to these parameters. Please use New-TimeSpan cmdlet object to construct Timespan variables. Please refer [New-TimeSpan](/powershell/module/microsoft.powershell.utility/new-timespan) to know more about New-TimeSpan.
 - Parameter `-ResourceGroupName` would no longer support alias `-ResourceGroup`.
+- Parameter 'Namespace' has been renamed to 'NamespaceName'.
```

Comment:
Replace single quote (') with (`) for parameter names so they are displayed with a grey background everywhere. @Saglodha